packages/cms-cli/commands/logs.js

```diff
@@ -68,6 +72,14 @@ const loadAndValidateOptions = async options => {
   }
 };
 
+const logError = async (error, portalId, functionPath) => {
+  return logServerlessFunctionApiErrorInstance(
+    error,
+    await getScopeDataForFunctions(portalId),
```

Comment:
Ideally, this could go within `errorHandlers.js` but it creates a circular dependency there.